src/main.js

```diff
@@ -0,0 +1,198 @@
+'use strict';
+// Check email validity
+const emailInput = document.querySelector('#email');
+const failureMsgNull = document.querySelector('.failure-message-null');
+const failureMsgInvalid = document.querySelector('.failure-message-invalid');
+const checkEmailValidity = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+
+emailInput.classList.remove('border-error');
+function emailChecker() {
+    const emailValue = emailInput.value;
+
+    if (emailValue.length < 1) {
+        failureMsgNull.classList.remove('hide');
+        emailInput.classList.add('border-error');
+    } else if (checkEmailValidity.test(emailValue) === false) {
```

Comment:
```suggestion
    } else if (!checkEmailValidity.test(emailValue)) {
```
이런식으로 보다 간단히 표현할수도있답니다!